src/components/test-category-icon/test-category-icon.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'test-category-icon',
+  templateUrl: 'test-category-icon.html'
+})
+export class TestCategoryIconComponent {
+  @Input()
+  category: string;
+
+  categoryIcons = {
+    'A': 'assets/icon/test-categories/A.png',
```

Comment:
Would it be better to have something like `iconPath` coming from config and then a function here to resolve the filename, e.g.
`(category) => \`${iconPath}/${category}.png\``

Reply:
We aren't going to have 1 icon per category, once we get the full list each icon will be used multiple times, so unless we duplicate the images this wouldn't work